### PR TITLE
[DPEDE-608] Include chi-tabs wc in documentation, add inverse property

### PR DESCRIPTION
**Pull request by ASanchLumen**
https://ctl.atlassian.net/browse/DPEDE-608

### CODE REVIEW

File: src/website/views/components/tabs/_examples.pug [hunk 1]
```diff
@@ -587,6 +729,10 @@ h2 Examples
 
         <script>chi.tab(document.getElementById('example-vertical-solid'));</script>
 
+
+//- 
+//-  ENTERPRISE PORTAL
+//-
```
Comment (josecarlosgonzalezv) [inline src/website/views/components/tabs/_examples.pug:735]:
```suggestion
```

File: src/website/views/components/tabs/_examples.pug [hunk 2]
```diff
@@ -58,23 +58,46 @@ h2 Examples
...
               href=`#html-base`
               role="tab"
               aria-selected="true"
               aria-controls="html-base"
               ) HTML Blueprint
-      .chi-tabs-panel#webcomponent-base(role="tabpanel")
-      .chi-tabs-panel.-active#html-base(role="tabpanel")
+      .chi-tabs-panel.-active#webcomponent-base(role="tabpanel")
+        :code(lang='html')
+          <chi-tabs
```
Comment (josecarlosgonzalezv) [inline src/website/views/components/tabs/_examples.pug:78]: Could we put `<chi-tabs>` just in one line? Same for every code snippets.

File: src/website/views/components/tabs/_examples.pug [hunk 3]
```diff
@@ -58,23 +58,46 @@ h2 Examples
               .-text Tab 3 content
       .example-tabs.-pl--2
         ul.chi-tabs#example-base(role="tabpanel")
-          li.-disabled
+          li.-active
```
Comment (josecarlosgonzalezv) [inline src/website/views/components/tabs/_examples.pug:61]: You have added just the code snippets, should we change our current examples with the web components as well? @jllr

File: src/website/views/components/tabs/_examples.pug [hunk 4]
```diff
@@ -58,23 +58,46 @@ h2 Examples
...
-      .chi-tabs-panel#webcomponent-base(role="tabpanel")
-      .chi-tabs-panel.-active#html-base(role="tabpanel")
+      .chi-tabs-panel.-active#webcomponent-base(role="tabpanel")
+        :code(lang='html')
+          <chi-tabs 
+            id="chi-tabs-example" 
+            active-tab="chi-tabs-example-tab-1" 
+            sliding-border
+          ></chi-tabs>
+          <script>
```
Comment (josecarlosgonzalezv) [inline src/website/views/components/tabs/_examples.pug:83]: Space between blocks
```suggestion
          ></chi-tabs>
          
          <script>
```

File: test/chi/custom-elements/tabs.pug [hunk 5]
```diff
@@ -12,8 +12,90 @@ h2 Base
...
+.-w--100 
+  chi-tabs(id='size-SM-horizontal' size='sm')
+h4 MD
+.-w--100 
+  chi-tabs(id='size-MD-horizontal' size='md')
+h4 LG
+.-w--100 
+  chi-tabs(id='size-LG-horizontal' size='lg')
+h4 XL
+.-w--100
```
Comment (josecarlosgonzalezv) [inline test/chi/custom-elements/tabs.pug:75]: I think `xl` and `sm--2` are not supported in tabs, am I right @jllr ?

**Review by josecarlosgonzalezv — CHANGES_REQUESTED** [review]

File: test/chi/custom-elements/tabs.pug (hunk 5 above)
Comment (ASanchLumen) [reply test/chi/custom-elements/tabs.pug:75]: Good catch!  Changed "sm--2" for "sm". 

`xl` is supported for tabs in `constants/size.ts`

**Review by jllr — APPROVED** [review]

**Review by josecarlosgonzalezv — APPROVED** [review]

**Pull request merged**